recipes-app/iot2050-event-record/README.md

```diff
@@ -0,0 +1,71 @@
+# IOT2050 Event Record
+
+IOT2050 Event Record is using for reading and recording events, such as
+power up, power loss, tilted, uncovered, watchdog reset and eio events.
+
+The core is a RPC service implemented with the help of gPRC.
+
+In addition:
+- A event-serve service for writing and reading syslog.
+- A event-record service for recording events.
+
+# Sensor events
+
+In default, the sensor events, i.e. tilted and uncovered events, are disabled.
+If enabling logging of sensor events is expected, please create a systemd
+drop-in for `iot2050-event-record.service`, as follows:
+
+```sh
+cp /usr/lib/iot2050/event/iot2050-event-record.conf /etc/systemd/system/iot2050-event-record.service.d/
+```
+
+# Watchdog events example
+
+If watchdog events recording is expected, please refer to [WATCHDOG.md](./WATCHDOG.md).
```

Comment:
It's not `example` anymore

Reply:
Do we have more than 1 event for watchdog?

Reply:
ACK and done.